packages/web-frontend/src/screens/mobile/RootScreen/index.js

```diff
@@ -72,11 +72,23 @@ const RootScreen = ({
     setSelectedTab,
   ]);
 
+  // maintain the header height in state, so the overlay div can sit below it
```

Comment:
It makes sense to use useRef() here. While I was trying things, I've removed the `useEffect` and the overlay still sit below the header. Just wondering what will make it overlapped with header?

Reply:
You're such a good reviewer going to that effort! This isn't used to make sure the overlay sits below the header, but that the overlay fills the remaining space on the screen, but without creating a scrollbar. 
![image](https://user-images.githubusercontent.com/1274422/158713901-ab93c93f-64c5-4423-950e-18af43037e3a.png)
This was actually a little tricky to get right, especially on safari for iOS, so this is a little bit of a compromise solution. There were a couple of options but they all seemed worse:
- Have it always set to `min-height: 100vh;`. This would make sure it filled the screen area, but would overfill it and create a scrollable area, which is a bit weird (though much much simpler code, so I almost went with it)
- Use a react portal here which fills the space, and render the library into that. I tried this too, and it felt like overkill - involved more complex code that what we end up with here